doc/CONNECTION_MANAGER.md

```diff
@@ -18,3 +18,39 @@ The following is a list of available options for setting limits for the Connecti
 - `pollInterval`: sets the poll interval (in milliseconds) for assessing the current state and determining if this peer needs to force a disconnect. Defaults to `2000` (2 seconds).
 - `movingAverageInterval`: the interval used to calculate moving averages (in milliseconds). Defaults to `60000` (1 minute). This must be an available interval configured in `Metrics`
 - `defaultPeerValue`: number between 0 and 1. Defaults to 1.
+- `gater`: gater options.
+
+### Gater Options
```

Comment:
It may be nice to have the comments from https://github.com/libp2p/go-libp2p-core/blob/master/connmgr/gater.go#L11-L43 included here or wherever is most appropriate